### PR TITLE
Changelog and version updates for release v0.25.0

### PR DESCRIPTION
**Pull request by eikenb**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,50 @@
...
+## v0.25.0 (Apr 27, 2020)
 
 IMPROVEMENTS:
 
-* Add minimum and maximum math functions
+* Add minimum and maximum math functions [[GH-1323](https://github.com/hashicorp/consul-template/pull/1323)]
+
+* Allow overriding the default delimiter [[GH-1290](https://github.com/hashicorp/consul-template/pull/1290)]
+
+* Add weights field for HealthService [[1288](https://github.com/hashicorp/consul-template/pull/1288)]
```
Comment (lornasong) [inline CHANGELOG.md:9]: recommend "[1288]" => "[GH-1288]"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,50 @@
...
 BUG FIXES:
 
-- Fix issue reading kv2 vault secrets for key paths starting with "data" [[GH-1340](https://github.com/hashicorp/consul-template/issues/1340)]
+* Fail when kv2 secret path starts with "data" [[GH-1341](https://github.com/hashicorp/consul-template/pull/1341), [GH-1340](https://github.com/hashicorp/consul-template/issues/1340)]
+
+* Don't renew vault token when no token is set [[GH-1352](https://github.com/hashicorp/consul-template/pull/1352), [GH-1297](https://github.com/hashicorp/consul-template/issues/1297)]
+
+* Fix bug looking up versioned vault secrets [[GH-1354](https://github.com/hashicorp/consul-template/pull/1354), [GH-1350](https://github.com/hashicorp/consul-template/issues/1350)]
+
+* Fix issue reading kv2 vault secrets for key paths starting with "data" [[GH-1340](https://github.com/hashicorp/consul-template/issues/1340), [GH-1341](https://github.com/hashicorp/consul-template/pull/1341)]
```
Comment (lornasong) [inline CHANGELOG.md:34]: I see this change twice, also at line 28 as:
> Fail when kv2 secret path starts with "data"

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,14 +1,50 @@
...
+
+* Fix data race in child.go [[GH-1372](https://github.com/hashicorp/consul-template/issues/1372)]
```
Comment (lornasong) [inline CHANGELOG.md:36]: just double checking that leaving out the pull request link is intentional

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,14 +1,50 @@
...
+
+* Fix issue with template commands when used as library [[GH-1370](https://github.com/hashicorp/consul-template/pull/1370), [GH-1369](https://github.com/hashicorp/consul-template/issues/1369)]
+
+DOCUMENTATION:
+
+* Howto run multiple thinngs in template commands [[GH-1375](https://github.com/hashicorp/consul-template/pull/1375)]
```
Comment (lornasong) [inline CHANGELOG.md:42]: suggestion to change "Howto" => "How to" and "thinngs" => "things"

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,14 +1,50 @@
...
+
+* Template command_timeout clarifications [[GH-1369](https://github.com/hashicorp/consul-template/issues/1369)]
```
Comment (lornasong) [inline CHANGELOG.md:44]: just double checking that leaving out the pull request link is intentional

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,14 +1,50 @@
...
+
+* ByMeta does not accept `services` as input [[GH-1348](https://github.com/hashicorp/consul-template/issues/1348)]
```
Comment (lornasong) [inline CHANGELOG.md:46]: looks like you resolved this with a commit. double-checking if you would want to include that link?

File: CHANGELOG.md (hunk 1 above)
Comment (eikenb) [reply CHANGELOG.md:9]: Yep. Thanks.

File: CHANGELOG.md (hunk 2 above)
Comment (eikenb) [reply CHANGELOG.md:34]: Kept wording from this one as it was clearer. Thanks.

File: CHANGELOG.md (hunk 4 above)
Comment (eikenb) [reply CHANGELOG.md:42]: thinnnnnnnnngs

File: CHANGELOG.md (hunk 5 above)
Comment (eikenb) [reply CHANGELOG.md:44]: Guess I should have both everywhere. Fixed.

File: CHANGELOG.md (hunk 6 above)
Comment (eikenb) [reply CHANGELOG.md:46]: Yep. Want the link to the issue. Keeping as is. Thanks.

File: CHANGELOG.md (hunk 3 above)
Comment (eikenb) [reply CHANGELOG.md:36]: Nope, didn't mean to. Fixed.

File: CHANGELOG.md (hunk 6 above)
Comment (lornasong) [reply CHANGELOG.md:44]: sorry! just double checking I was being clear: i meant do you want to add the additional commit link (in lieu of the pr link)

**Comment by eikenb** [comment]
merged and pushed locally

**Pull request closed without merging**